y_pef_schedule.module

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @file
+ * Implements hooks for the y_pef_schedule module.
+ */
+
+declare(strict_types = 1);
```

Comment:
Is it something new in the PHP files?
Havent seen it is used in Drupal code.

Reply:
It seems it came with PHP 7 and is recommended for use everywhere.